sklearn/linear_model/tests/test_ransac.py

```diff
@@ -353,3 +353,14 @@ def test_ransac_dynamic_max_trials():
     ransac_estimator = RANSACRegressor(base_estimator, min_samples=2,
                                        stop_probability=1.1)
     assert_raises(ValueError, ransac_estimator.fit, X, y)
+
+def test_ransac_fit_sample_weight():
+    ransac_estimator = RANSACRegressor()
```

Comment:
you should set random_state to be some number to get the same results, sampling every time.